src/components/CatalogOverlay/CatalogOverlayPlotSettingsPanelComponent/CatalogOverlayPlotSettingsPanelComponent.tsx

```diff
@@ -172,52 +172,54 @@ export class CatalogOverlayPlotSettingsPanelComponent extends React.Component<Wi
                 <FormGroup label={"Scaling"} inline={true} disabled={disableSizeMap}>
```

Comment:
Perhaps we also include the Scaling dropdown in the Collapse element.

Reply:
To make the collapse behavior consistent, I moved and duplicated FormGroup `label="Size min"` and `label="Size max"` from `const sizeMap` to `const sizeMajor` and `const sizeMinor`